tests/raftstore_cases/test_split_region.rs

```diff
@@ -153,9 +153,9 @@ fn test_server_split_region_twice() {
     let c = Box::new(move |write_resp: WriteResponse| {
         let mut resp = write_resp.response;
         let admin_resp = resp.mut_admin_response();
-        let split_resp = admin_resp.mut_split();
-        let left = split_resp.take_left();
-        let right = split_resp.take_right();
+        let split_resp = admin_resp.mut_splits();
+        let mut regions = split_resp.take_regions();
+        let (right, left) = (regions.pop().unwrap(), regions.pop().unwrap());
```

Comment:
`let (left, right) = (regions[0], regions[1])` looks more intuitive.

Reply:
I like it. However rust doesn't work that way.